### ISSUE TITLE
Customer approval duplicate buttons

### ISSUE THREAD

**Issue by condor2** [opencart/opencart#11059]
**What version of OpenCart are you reporting this for?**
Latest github

Duplicated buttons on Customer approval
![custom_approv_butt](https://user-images.githubusercontent.com/14251939/172302849-f0f1f3ac-4f3f-45b2-8a49-92f11e731959.png)

**Comment by danielkerr** [comment]
what duplicate buttons? dont post such rubish!

**Comment by danielkerr** [comment]
yes because they should have the option of doing either depending if they need to mass approve

**Comment by danielkerr** [comment]
its dumb to post small crap like this. its clear to most the reason why

**Comment by danielkerr** [comment]
It's not a bug! Just wasting my time
